app/models/person.rb

```diff
@@ -1373,6 +1373,22 @@ def alive_status
     @alive_status = verification_types.where(type_name: VerificationType::ALIVE_STATUS).first
   end
 
+  # In order to avoid adding another callback,
+  # this method is used after consumer_role is created for person
+  # 1.) Factories::EnrollmentFactory -> self.build_consumer_role
+  #   - used for consumer sign-up
+  # 2.) Family -> build_consumer_role
+  #   - used when creating a family_member from the 'Manage Family' from a user's homepage
+  # 3.) Insured::FamiliesHelper -> build_consumer_role
+  #   - used as a failsafe in case any family_members are missing a consumer role
+  # 4.) Operations::People::CreateOrUpdateConsumerRole -> create_consumer_role
+  #   - used when the legacy ::FinancialAssistance::Operations::Families::CreateOrUpdateMember propagates an applicant to a family_member
+  def create_demographics_group
+    return unless demographics_group.blank? && consumer_role.present?
+    demographics_group = DemographicsGroup.new(alive_status: AliveStatus.new)
```

Comment:
Create alive status in a different method, and check the presence of the alive status before create a new one.